### ISSUE TITLE
`babel-plugin-superjson-next`: The produced js should target ES2015 like Next.js default

### ISSUE THREAD

**Issue by baptisteArno** [blitz-js/superjson#148]
Hi!

It looks like the produced file is not ES2015 code. Any way I can enable this?

Shouldn't be the default behavior as Next.js is targetting ES2015 by default?

**Comment by Skn0tt** [comment]
Hi @baptisteArno! SuperJSON can be used outside of Next.js, which is why we need to be careful around such changes. Could you elaborate / share a link on the implications of targeting ES2015? What are the improvements that would bring?

**Comment by baptisteArno** [comment]
Sure I understand, it basically allows targeting IE11 browser.

So, no option to do this as of now?

**Comment by Skn0tt** [comment]
No, there's no way for you to change the build output for SuperJSON locally. I could definitely see us moving to ES2015 output, if there are considerable improvements in that. If it does nothing but remove IE11 support, that'd be a net-negative.
